Makefile

```diff
@@ -737,21 +737,39 @@ $(LINK_DATA): $(wildcard lib/*.js) tools/doc/apilinks.mjs | out/doc
 $(VERSIONS_DATA): CHANGELOG.md src/node_version.h tools/doc/versions.mjs
 	$(call available-node, tools/doc/versions.mjs $@)
 
+node_use_icu = $(call available-node,"-p" "process.versions.icu != undefined")
```

Comment:
Non-blocking, but maybe this could match one of the examples for detection in https://github.com/nodejs/node/blob/master/doc/api/intl.md#detecting-internationalization-support?

Reply:
Good idea. Done! Thanks.